plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotMetadata.java

```diff
@@ -63,4 +75,199 @@ public void testTables()
                 Optional.empty());
         assertThat(withUppercaseTable.tableName()).isEqualTo("airlineStats");
     }
+
+    private void assertPQLEquals(Call call, String pql)
+    {
+        StringBuilder pqlBuilder = new StringBuilder();
+        PinotMetadata.buildConstraintPql(call, pqlBuilder);
+        assertThat(pqlBuilder.toString()).isEqualTo(pql);
+    }
+
+    private Call getCall(String functionName, String argType)
```

Comment:
Helper method should be located after the caller. Also, these method can be static.

Reply:
fixed